README.md

```diff
@@ -3,6 +3,11 @@
 A Vue.js component library.
 
 ## 🚀 Install
+
 ```sh
+# install maker
 npm i @square/maker
+
+# install peer dependencies
+npx i-peers -a
```

Comment:
FYI this doesn't install the peer dependencies of @square/maker. It installs the peer dependencies defined in the current `package.json`.

Reply:
oh, I thought `i-peers -a` recursively installed all peer deps... then how do I achieve the latter with a single command?